CRM/Civioffice/Form/LocalDocumentStore/LocalDocumentStoreSettings.php

```diff
@@ -63,47 +63,60 @@ public function buildQuickForm()
         parent::buildQuickForm();
     }
 
-    /**
-     * Validate input data
-     * @return bool
-     */
-    public function validate(): bool
-    {
-        parent::validate();
+  /**
+   * Validate input data
+   *
+   * @return bool
+   */
+  public function validate(): bool {
+    parent::validate();
 
-        // verify that the folder is 1) there, 2) readable
-        if (!empty($this->_submitValues['local_folder'])) {
-            $local_folder = $this->_submitValues['local_folder'];
-            if (!file_exists($local_folder) && !mkdir($local_folder, 0777, true)) {
-              $this->_errors['local_folder'] = E::ts('Could not create directory');
-            } else if (!is_dir($local_folder)) {
-                $this->_errors['local_folder'] = E::ts("This is not a folder");
-            } else if (!is_readable($local_folder)) {
-                $this->_errors['local_folder'] = E::ts("This folder cannot be accessed");
-            }
+    // verify that the folder is 1) there, 2) readable
+    if (!empty($this->_submitValues['local_folder'])) {
+      $local_folder = trim($this->_submitValues['local_folder']);
+      if (!file_exists($local_folder) && !mkdir($local_folder, 0777, TRUE)) {
+        $this->_errors['local_folder'] = E::ts('Could not create directory');
+      }
+      else {
+        if (!is_dir($local_folder)) {
+          $this->_errors['local_folder'] = E::ts('This is not a folder');
         }
-
-        if (!empty($this->_submitValues['local_temp_folder'])) {
-            $local_temp_folder = $this->_submitValues['local_temp_folder'];
-            if (!file_exists($local_temp_folder) && !mkdir($local_temp_folder, 0777, true)) {
-              $this->_errors['local_temp_folder'] = E::ts('Could not create directory');
-            } else if (!is_dir($local_temp_folder)) {
-                $this->_errors['local_temp_folder'] = E::ts("This is not a folder");
-            } else if (!is_readable($local_temp_folder)) {
-                $this->_errors['local_temp_folder'] = E::ts("This folder cannot be accessed");
-            }
+        else {
+          if (!is_readable($local_folder)) {
```

Comment:
The `if` could be combined with the `else` above to `elseif` to have one block less.